blockchain/config.go

```diff
@@ -32,6 +32,9 @@ type (
 		BloomfilterIndexDBPath string           `yaml:"bloomfilterIndexDBPath"`
 		CandidateIndexDBPath   string           `yaml:"candidateIndexDBPath"`
 		StakingIndexDBPath     string           `yaml:"stakingIndexDBPath"`
+		SGDIndexDBPath         string           `yaml:"sgdIndexDBPath"`
+		SGDIndexCacheSize      int              `yaml:"sgdIndexCacheSize"`
+		SGDPercentage          uint64           `yaml:"sgdPercentage"`
```

Comment:
no need for percentage, we'll have a tiered reward structure, like 50% for 100k volume, 60% for 200k volume, etc.